grep/tests/test.bats

```diff
@@ -0,0 +1,11 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Version matches" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} grep --version)"
```

Comment:
If you want to test the full output, you should use:

```
@test "Version matches" {
  run hab pkg exec ${TEST_PKG_IDENT} grep --version
  [ "$output" = "grep (GNU grep) "${TEST_PKG_VERSION}" ]
}
```

Reply:
Fixed!